wisp-ssl/src/main/kotlin/wisp/security/ssl/PemComboFile.kt

```diff
@@ -69,10 +69,8 @@ data class PemComboFile(
                             Regex("-+END PRIVATE KEY-+")
                         )
                     }
-                    line.isBlank() -> {
-                        // This is ok, just keep going
-                    }
-                    else -> throw IOException("unexpected line: $line")
+
+                    // Ignore everything else
```

Comment:
Currently, it would fail if there are lines not recognized even though the cert is still valid so I think this should be fine we don't need to verify if a certificate is actually parsed or not since the old logic allows an empty cert?